classes/driver/field/checkbox.php

```diff
@@ -6,6 +6,9 @@
 
 class Driver_Field_Checkbox extends Driver_Field_Abstract
 {
+    use Trait_Driver_Field_Html_Attributes;
```

Comment:
Using traits makes this app not compatible with php 5.3. This is not a problem, but it should be noticed somewhere.